.wordlist.txt

```diff
@@ -184,6 +184,8 @@ Keychain
 KiB
 kickstart
 kickstarts
+Killswitch
+killswitch
```

Comment:
The lower-case version of words is generally sufficient as it's automatically allowed that any word starts with a capital letter, e.g. to cover the case when they start a sentence.